Cartfile

```diff
@@ -2,3 +2,4 @@ github "jdg/MBProgressHUD" ~> 0.9.2
 github "cbpowell/MarqueeLabel" ~> 3.1.4
 github "pixelogik/ColorCube" "master"
 github "dennisweissmann/DeviceKit" ~> 1.7
+github "GianniCarlo/SwiftReorder"
```

Comment:
I had considered it, but I ran into a problem making the original repo Carthage compatible, since I wanted to try it right away, I brute forced it in my fork recreating the whole project and just adding the library files without the example. There's an improvement besides the addition of the hover/drop callback events, in the original the calculation for the reorder was taking into consideration all the visible cells, when it's only necessary to take into account the adjacent cells. I'll get around to it someday 🤔